src/Page/Block/ArticleContent.php

```diff
@@ -6,24 +6,23 @@
 use App\Page\View;
 use Symfony\Component\OptionsResolver\OptionsResolver;
 
-class ArticleContent implements Block
+class ArticleContent extends Block
 {
     use WithApiClient;
 
-    public const ID = 'id';
-
-    public function __invoke(array $options): array|View
+    public function __invoke(int $id): array|View
     {
-        $data = $this->apiClient->get('/v2/articles/'.$options['id']);
+        $data = $this->apiClient->get('/v2/articles/'.$id);
 
         return new View\ArticleContent($data['item']);
     }
 
     public function configureOptions(OptionsResolver $resolver): void
```

Comment:
The option resolver is not necessary. Name, type and default value are defined in the signature of `__invoke`.